tests/test_stl_binders.py

```diff
@@ -309,3 +309,25 @@ def test_map_delitem():
     del um["ua"]
     assert sorted(list(um)) == ["ub"]
     assert sorted(list(um.items())) == [("ub", 2.6)]
+
+
+def test_map_view_types():
+    map_string_double = m.MapStringDouble()
+    unordered_map_string_double = m.UnorderedMapStringDouble()
+    map_string_double_const = m.MapStringDoubleConst()
+    unordered_map_string_double_const = m.UnorderedMapStringDoubleConst()
+
+    keys_type = type(map_string_double.keys())
```

Comment:
Do these also subclass the collections abc types? https://docs.python.org/3/library/collections.abc.html I am not sure we can do that via a mixin, but if we could that would be ideal.

Reply:
I don't think so... Those have a lot of extra functionality we don't support right now, don't they?

Reply:
OK, after checking it a bit more thoroughly, it definitly requires some extra functionality that is not currently implemented.

The main thing is that both `KeysView` and `ItemsView` inherit from `collections.abc.Set`, and should therefore implement `__le__`, `__lt__`, `__eq__`, `__ne__`, `__gt__`, `__ge__`, `__and__`, `__or__`, `__sub__`, `__xor__`, and `isdisjoint`.

I would definitely like to add that support, but IMHO it should probably be a separate PR.